src/main/scala/net/ceedubs/ficus/readers/DurationReaders.scala

```diff
@@ -13,8 +13,8 @@ trait DurationReaders {
    */
   implicit def finiteDurationReader: ValueReader[FiniteDuration] = new ValueReader[FiniteDuration] {
     def read(config: Config, path: String): FiniteDuration = {
-      val millis = config.getDuration(path, java.util.concurrent.TimeUnit.MILLISECONDS)
-      FiniteDuration(millis, MILLISECONDS)
+      val nanos = config.getDuration(path, NANOSECONDS)
+      Duration.fromNanos(nanos)
```

Comment:
Another option is to avoid the java duration altogether and do

```scala
val str = config.getString(path)
Duration(str)
```

Although this returns a `Duration` instead of a `FiniteDuration`